publish-wrims-template/build.gradle

```diff
@@ -0,0 +1,102 @@
+plugins {
+    id 'maven-publish'
+    id 'distribution'
+}
+
+def zipFilePath = project.findProperty('wrimsGuiZipFile') ?: 'C:/WRIMS/wrims_gui_template.zip'
+def templateCoreFile = project.findProperty('wrimsGuiZipFile') ?: 'C:/WRIMS/wrims_gui_template_core.zip'
+
+repositories {
+    maven {
+        name = "GitHubPackages"
+        url = uri("https://maven.pkg.github.com/CentralValleyModeling/wrims")
+        credentials {
+            username = project.findProperty('githubToken') ?: System.getenv('GITHUB_TOKEN')
+            password = ''
+        }
+    }
+}
+
+configurations {
+    testFile
+}
+
+publishing {
+    publications {
+        wrimsGuiTemplateCore(MavenPublication) {
+            groupId 'gov.ca.dwr'
+            artifactId 'wrims_gui_template_core'
+            version = '1.0.0'
+            artifact(templateCoreFile) {
+                extension 'zip'
+            }
+            pom {
+                name = 'WRIMS GUI Template Core'
+                description = 'Template for WRIMS GUI'
+                url = 'https://github.com/CentralValleyModeling/wrims'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                scm {
+                    connection = 'scm:git:git://github.com/CentralValleyModeling/wrims.git'
+                    developerConnection = 'scm:git:ssh://github.com/CentralValleyModeling/wrims.git'
+                    url = 'https://github.com/CentralValleyModeling/wrims'
+                }
+            }
+        }
+//        wrimsGuiTemplate(MavenPublication) {
```

Comment:
404 template does not exist....

If this is still work in progress, the PR should get marked as draft.

Reply:
Josh deleted subproject